portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
I don't understand why all the civic code resides in OncoKBConnector.js? Is it going to be different in the new version of PV? cc'ing @zhx828 @onursumer